src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -81,7 +68,7 @@ class LSFJobManager extends AbstractLSFJobManager {
      *    * Short (with status flag)
      *    * Long (with status flag)
      */
-    ZonedDateTime parseTime(String str) {
+    ZonedDateTime parseTime(String str, ZonedDateTime referenceDate = ZonedDateTime.now()) {
```

Comment:
Needed to fix the test (which otherwise fails between 1.1. and mid February every year :-D)